test/acceptance/features/bindAppToService.feature

```diff
@@ -512,10 +513,14 @@ Feature: Bind an application to a service
                     version: v1
                     kind: Backserv
                     name: demo-backserv-cr-2
+                application:
+                    name: gen-app-a-s-d
+                    group: apps
+                    version: v1
+                    resource: deployments
```

Comment:
this scenario should be now either renamed or removed, because adding the application references changes the scenario semantic.

Reply:
This is a valid scenario. The empty application was used earlier for convenience.

Reply:
Similar to above - by introducing now application ref, the usecase is not the one described by the scenario anymore:

"Creating binding secret from the definitions managed in OLM operator descriptors"

We do not just create secret anymore, we inject binding data into the application. That should be reflected into the scenario description and the appropriate assertions.

Reply:
Done. 29d348dd24be04f7ef01892df9186f888247adf1